targets/unity-v2/Testing/Tests/PersistentSockets/PubSubTest.cs

```diff
@@ -11,14 +11,30 @@ public class PubSubTest : UUnitTestCase
     {
         static PlayFab.EventsModels.EntityKey _MyEntityKey = null;
         static string _previousTitleId = null;
+        public override void SetUp(UUnitTestContext testContext)
+        {
+            // specific title id for relay test
+            _previousTitleId = PlayFabSettings.TitleId;
+            PlayFabSettings.TitleId = "70B02F89";
+            PlayFabSettings.VerticalName = "spi-relay";
+        }
 
         public override void Tick(UUnitTestContext testContext)
         {
             // No async work needed?
             // should this be a theoretical HubConnection SendAsync call to print a message out in the test?
         }
+
         public override void TearDown(UUnitTestContext testContext)
         {
+            PlayFabSettings.TitleId = _previousTitleId;
+            PlayFabSettings.VerticalName = null;
+
+            // if test didn't complete, fail
```

Comment:
This is unnecessary.
It's literally saying "if the test failed, then fail the test"
You generally shouldn't change the state of the test in teardown.